tests/libvmifact/factory.go

```diff
@@ -76,7 +76,14 @@ func NewAlpine(opts ...libvmi.Option) *kvirtv1.VirtualMachineInstance {
 		libvmi.WithRng(),
 	}
 	alpineOpts = append(alpineOpts, opts...)
-	return libvmi.New(alpineOpts...)
+	vmi := libvmi.New(alpineOpts...)
+
+	// user-data e2e tests require cloud init volume
```

Comment:
What tests are we talking about here?

Reply:
These tests https://github.com/kubevirt/kubevirt/blob/main/tests/vmi_lifecycle_test.go#L370

Reply:
Shouldn't we explicitly add `libvmi.WithCloudInitNoCloud` there then?